src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderEclipseDefaultPositive.java

```diff
@@ -19,19 +19,19 @@
 import static javax.swing.WindowConstants.*;
 import static org.junit.Assert.assertEquals;
 
-import java.awt.Button; // violation
-import java.awt.Dialog; // violation
-import java.io.InputStream; // violation
+import java.awt.Button; // violation 'Import statement .*Button' is in the wrong order. Should be in the .* group, expecting not assigned imports.*'
```

Comment:
Can it be `wrong order. Should be in the .* group, expecting not assigned imports.*` ?

No multiple Imports on same line, so we can skip what exact import we have in message.
Apply to all others.

Reply:
ok. it can be `.* wrong order. Should be in the .*group, expecting not assigned imports.*`.

Reply:
done